pkg/etcdenvvar/etcd_env.go

```diff
@@ -316,19 +317,17 @@ func getEtcdEndpoints(configmapLister corev1listers.ConfigMapLister, skipBootstr
 	bootstrapAddress := etcdEndpoints.Annotations["alpha.installer.openshift.io/etcd-bootstrap"]
 	// In some cases we want to exclude the ephemeral bootstrap-etcd member from the endpoint list.
 	if !skipBootstrap && len(bootstrapAddress) > 0 {
-		ip, err := dnshelpers.GetURLHostForIP(bootstrapAddress)
-		if err != nil {
+		if ip := net.ParseIP(bootstrapAddress); ip == nil {
```

Comment:
@tjungblu  why did we replace `GetURLHostForIP` here ?

Reply:
that's from https://github.com/openshift/cluster-etcd-operator/pull/820/files#diff-6d891f4c2f7afdeba2df319982323725c7967e877d896353908b8e660673bb9cR320

I reckon that the net.ParseIP is better than the home-grown solution with string concat we had before. @stbenjam can surely also answer.

Reply:
GetURLHostForIP was replaced with the use of  `net.JoinHostPort` below, which comes from the standard library.  There's also a not-so-intelligent linter I'm working on adding that can find incorrect URL constructions (which break IPv6), but it requires using JoinHostPort to work.